samza-azure/src/main/java/org/apache/samza/BarrierState.java

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
Quick comment on the package organization of all source code files in samza-azure. Please put the classes into a proper sub-package name, instead of clubbing them all under org.apache.samza. For example, AzureJobCoordinator definitely should go to package org.apache.samza.coordinator